app/controllers/casa_admins_controller.rb

```diff
@@ -39,10 +39,11 @@ def create
     authorize @casa_admin
 
     begin
-      service.create!
+      casa_admin = service.create!
+      notice_msg = send_sms casa_admin.phone_number, "admin"
```

Comment:
- I'd split these out. Just so the SMS is sent in one line and the notice/resource stuff is its own thing on another line

Reply:
mmm actually can u check if what I did with the splitting is what u had in mind? I'm not too sure what the benefit of this split is. To me, it just seems more code.

Reply:
I like this it works too. It was just a clarity thing. Your status additions improve said clarity.